pex/third_party/__init__.py

```diff
@@ -116,8 +116,11 @@ def iter_root_packages(self, relpath):
       yield package
 
   def _filter_names(self, relpath, pattern, group):
+    # We use '/' here instead of os.sep because the zip file format spec specifies that paths must use forward slashes.
+    # See section 4.4.17 of https://support.pkware.com/display/PKZIP/APPNOTE
```

Comment:
It looks like this link works a bit better: https://pkware.cachefly.net/webdocs/casestudies/APPNOTE.TXT.